### PR TITLE
Daily recurring cronjob for the bq-indexer

### PR DESCRIPTION
**Pull request by alahwa**
GKE has some neat extras for this, like metrics/visualizations, and features like pause/resume.

### CODE REVIEW

File: bigquery/deploy/bq-indexer.yaml.templ [hunk 1]
```diff
@@ -1,31 +1,36 @@
-apiVersion: batch/v1
-kind: Job
+apiVersion: batch/v1beta1
```
Comment (melissachang) [inline bigquery/deploy/bq-indexer.yaml.templ]: Have you confirmed this works? Maybe temporarily set to run every 5 mins.

Let me know what project you tested on, I want to see what this looks like in GCP console.

Comment (melissachang) [inline bigquery/deploy/bq-indexer.yaml.templ]: I don't want to make all users run a CronJob. Can you leave bq-indexer.yaml.templ/deploy-indexer.sh alone, and create bq-indexer-cronjob.yaml.templ with a comment on top:
```
Use this file to run the indexer in a cronjob.
```
I don't think it's worth mentioning anything anywhere else, like a README.

**Comment by alahwa** [comment]
Separated out to a separate file

File: bigquery/deploy/bq-indexer-cronjob.yaml.templ [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Use this config to run the indexer as a CronJob on a regular schedule
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: bq-indexer
+spec:
+  # Run daily
+  schedule: "*/3 * * * *"
```
Comment (melissachang) [inline bigquery/deploy/bq-indexer-cronjob.yaml.templ]: Change to once a day

Add comment like `Run every day at 5am UTC`

Comment (alahwa) [reply bigquery/deploy/bq-indexer-cronjob.yaml.templ]: Oops, good catch

**Pull request merged**